src/CreateMiddleware/CreateMiddleware.php

```diff
@@ -41,10 +41,11 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     /**
      * @param string $class
      * @param string|null $projectRoot
+     * @param string|null $classSkeleton
      * @return string
      * @throws CreateMiddlewareException
      */
-    public function process($class, $projectRoot = null)
+    public function process($class, $projectRoot = null, $classSkeleton = null)
```

Comment:
You can override the constant, but you have to use there `static::CONST` instead of `self::CONST`.

```php
class A {
    const MYCONST = 'A';

    public function x() {
        return static::MYCONST;
    }
}

class B extends A {
    const MYCONST = 'B';
}

$b = new B;
var_dump($b->x());
```

https://3v4l.org/9tmtn

```
Output for 5.6.0 - 5.6.30, hhvm-3.15.4 - 3.19.0, 7.0.0 - 7.2.0alpha2
string(1) "B"
```